examples/expressions/metadata/imp.rs

```diff
@@ -0,0 +1,81 @@
+use glib::prelude::*;
+use glib::subclass::prelude::*;
+use gtk::glib;
+
+use std::cell::RefCell;
+
+#[derive(Debug)]
+pub struct Metadata {
+    pub title: RefCell<String>,
+    pub last_modified: RefCell<glib::DateTime>,
+}
+
+#[glib::object_subclass]
+impl ObjectSubclass for Metadata {
+    const NAME: &'static str = "Metadata";
+    type Type = super::Metadata;
+    type ParentType = glib::Object;
+
+    fn new() -> Self {
+        let now = glib::DateTime::new_now_local().unwrap();
```

Comment:
very weird constructor name; cc @sdroege , maybe we should rename it to plain `now_local`?

Reply:
Sounds like a good idea, you do that? There are more similar constructors in datetime

Reply:
I did open an issue at least https://github.com/gtk-rs/gtk-rs-core/issues/342